modules/classic_ui/pages/index.adoc

```diff
@@ -16,3 +16,5 @@ To learn more about the ownCloud desktop and mobile clients, please refer to the
 * https://doc.owncloud.com/desktop/latest/[ownCloud Desktop Client]
 * https://doc.owncloud.com/android/latest[ownCloud Android App]
 * https://doc.owncloud.com/ios-app/latest[ownCloud iOS App]
+
+NOTE: Consider that this user guide covers the lates ownCloud version and the latest app versions only. This is because there would be too many dependencies and possibilites to cover. If you are using older versions, some items may not be present or look different.
```

Comment:
```suggestion
NOTE: this user guide covers the latest ownCloud version and the latest app versions only. This is because there would be too many dependencies and possibilities to cover. If you are using older versions, some items may not be present or look different.
```